docs/389ds/FAQ/bdb-deprecation.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "BDB backend deprecation"
+---
+
+# BDB Deprecation and using lmdb
+
+Since 389-ds-base 2.1.0, The 389 ldap directory server supports two kind of underlying database:
+
+- Berkeley Database (bdb)
+
+- Lightning Memory-Mapped Database Manager (lmdb)
+
+Starting from version 3.0.0 New instances are no more created with bdb by default but with libdb.
+
+Newly created instances are still created with bdb by default while libdb is flagged as [deprecated since Fedora 33]([Changes/Libdb deprecated - Fedora Project Wiki](https://fedoraproject.org/wiki/Changes/Libdb_deprecated)), this change is about to create lmdb instances by default.
+
+{% include toc.md %}
+
+## Why deprecating bdb backend ?
+
+Because the underlying library implementing the bdb version used by 389-ds-base is no more supported upstream and it is already deprecated in some OS:
+
+- [Changes/Libdb deprecated - Fedora Project Wiki](https://fedoraproject.org/wiki/Changes/Libdb_deprecated)
+
+## How to configure a new instance with lmdb or bdb ?
+
+There are several ways:
+
+1. Using dscreate --interactive:
+   Select the proper implementation when the following question is asked:
+
+   ```
+   Choose whether mdb or bdb is used.
+   ```
+
+2. Using dscreate --from-file
+   Set the following stanza in the template file:
+
+   ```
+   [slapd]
+   db_lib = mdb (or bdb)
+   ```
+
+3. Using dscreate with an environment variable.
+   Setting NSSLAPD_DB_LIB environment variable to 'bdb' or 'mdb' change the backend implementation chosen by default
+
+Set  db_lib = mdb (or bdb) in [slapd] section of dscreate template file
+
+## Can I mix different implementations ?
+
+Mixing implementaions limitations are:
+
+- The db_lib parameter is stored in the nsslapd-backend-implement attribute of cn=config entry, so it is global to the instance.
```

Comment:
```suggestion
- The `db_lib` parameter is stored in the `nsslapd-backend-implement` attribute of `cn=config` entry, so it is global to the instance.
```

Reply:
This attribute is rather in database config entry 'cn=config,cn=ldbm database,cn=plugins,cn=config'  not in 'cn=config'